sc2/unit.py

```diff
@@ -234,7 +234,7 @@ def has_vespene(self) -> bool:
     def weapon_cooldown(self) -> Union[int, float]:
         """ Returns some time (more than game loops) until the unit can fire again, returns -1 for units that can't attack
         Usage:
-        if not unit.weapon_cooldown:
+        if unit.weapon_cooldown == 0:
```

Comment:
any reason for this change?

Reply:
readabilty. I know this is the opposite of what i normally changed, but the cooldown is an actual number.